src/acc.rs

```diff
@@ -203,9 +355,42 @@ pub fn biophys(prop: &BiophysicalProperties, lems: &LemsFile) -> Result<Vec<Deco
     Ok(decor)
 }
 
-fn membrane(membrane: &MembraneProperties) -> Result<Vec<Decor>> {
+fn membrane(membrane: &MembraneProperties, inhomogeneous_parameters: &Map<String, ParsedInhomogeneousParameter>) -> Result<Vec<Decor>> {
     let known_ions = vec![String::from("ca"), String::from("k"), String::from("na")];
 
+    fn handle_ion_simple(known_ions: &Vec<String>, result: &mut Vec<Decor>, gs: &mut Map<String, String>, ion: &str, segmentGroup: &str, erev: &str) -> Result<()> {
+        if known_ions.contains(&ion.to_string()) {
```

Comment:
It's a copy of the code that was already there. First I duplicated it because the non-nernst mechanism and inhomogeneous mechanisms are very much the same, then moved the code that was equal between the two this function. But indeed, it looks like it needs some rethinking

Reply:
Let's keep it in; I'll massage it into shape later.